root/cgi-bin/print_ok.py

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+import os
+
+def print_ok():
+	print(f"Content-type: text/plain\n\n")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)